01-getting-started/getting-started.Rmd

```diff
@@ -5,4 +5,101 @@ output:
   html_notebook: 
     toc: true
     toc_float: true
----
\ No newline at end of file
+---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [About refine.bio](#about-refinebio)
+- [About how this tutorial book is structured](#about-how-this-tutorial-book-is-structured)
+- [What you need to install to run the examples](#what-you-need-to-install-to-run-the-examples)
+- [How to get the data](#how-to-get-the-data)
+- [How to use R Markdowns](#how-to-use-r-markdowns)
+- [An important note about file paths and Rmds](#an-important-note-about-file-paths-and-rmds)
+    - [Resources for learning R](#resources-for-learning-r)
+    - [More resources from the CCDL](#more-resources-from-the-ccdl)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## About refine.bio 
+
+[refinebio](https://www.refine.bio/about) is a collection of ready-to-use pre-processed normalized data! 
+Publicly available gene expression data is [uniformly processed](http://docs.refine.bio/en/latest/main_text.html#processing-information) and made available for easy download. 
+This tutorial has follow-along examples for use with refinebio downloads. 
+
+## About how this tutorial book is structured 
+
+This tutorial contains follow-along analysis examples for refinebio gene expression data.
+The analysis examples are organized by technology: ["microarray"](../02-microarray/intro-to-microarray.nb.html) or ["RNA-seq"](../03-rna-seq/intro-to-rna-seq.nb.html), in addition to an ["Advanced Topics"](../04-advanced-topics/intro-to-advanced-topics.nb.html) section. 
+Each analysis is self-contained and provides information with how to obtain the dataset used in the example from refinebio. 
```

Comment:
It may also be useful to adapt the [section of the README.md file](https://github.com/AlexsLemonade/refinebio-examples/blob/c47ce0b04c5049c879fee0ad2fe763bec76ad38c/README.md#how-to-use-this-repository) which outlines the contents of each module here. The section I am referring to can also be found below:

```
Each directory is a module of example workflow(s) and contains:

A README that introduces you to the concepts, requirements, and workflows for that module.
Example dataset(s) in the data folder
A results and/or plots folder(s) that contains the output of the analyses.
An R Notebook which consists of:
An R markdown (Rmd) file(s) that you can use in RStudio to run the analysis
An nb.html file that is the resulting output of the Rmd file rendered as an HTML file.
```

Reply:
Ah. Good idea. I like it.

Reply:
I'm not exactly sure what the READMEs will be during this revamping. I should file a discussion issue about this, because I've just realized I've only been thinking about this in my head and have not written it down.